model/aws/registry.yaml

```diff
@@ -481,3 +488,96 @@ groups:
+      - id: aws.sqs.queue.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the AWS SQS Queue the request refers to. Corresponds to the `--queue-name` parameter of the
+          SQS [get-queue-url](https://docs.aws.amazon.com/cli/latest/reference/sqs/get-queue-url.html) operation.
+        examples: ["some-sqs-queue-name"]
+  - id: registry.aws.sns
+    type: attribute_group
+    display_name: Amazon SNS Attributes
+    brief: >
+        This document defines attributes for AWS SNS.
+    attributes:
+      - id: aws.sns.topic.arn
+        type: string
+        stability: experimental
+        brief: >
+          The ARN of the AWS SNS Topic. An Amazon SNS [topic](https://docs.aws.amazon.com/sns/latest/dg/sns-create-topic.html) is a logical access point that acts as a communication channel. 
+        examples: ["arn:aws:sns:us-east-1:123456789012:mystack-mytopic-NZJ5JSMVGFIE"]
+  - id: registry.aws.kinesis
+    type: attribute_group
+    display_name: Amazon Kinesis Attributes
+    brief: >
+        This document defines attributes for AWS Kinesis.
+    attributes:
+      - id: aws.kinesis.stream_name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the AWS Kinesis [stream](https://docs.aws.amazon.com/streams/latest/dev/introduction.html) the request refers to. Corresponds to the `--stream-name` parameter of the
+          Kinesis [describe-stream](https://docs.aws.amazon.com/cli/latest/reference/kinesis/describe-stream.html) operation.
+        examples: ["some-stream-name"]
+  - id: registry.aws.stepfunctions
+    type: attribute_group
+    display_name: Amazon Step Functions Attributes
+    brief: >
+        This document defines attributes for AWS Step Functions.
+    attributes:
+      - id: aws.stepfunctions.activity.arn
+        type: string
+        stability: experimental
+        brief: >
+          The ARN of the AWS Step Functions Activity.
+        examples: ["arn:aws:states:us-east-1:123456789012:activity:get-greeting"]
+      - id: aws.stepfunctions.state_machine.arn
+        type: string
+        stability: experimental
+        brief: >
+          The ARN of the AWS Step Functions State Machine.
+        examples: ["arn:aws:states:us-east-1:123456789012:stateMachine:myStateMachine:1"]
+  - id: registry.aws.bedrock
+    type: attribute_group
+    display_name: Amazon Bedrock Attributes
+    brief: >
+        This document defines attributes for AWS Bedrock.
+    attributes:
+      - id: aws.bedrock.agent.id
+        type: string
+        stability: experimental
+        brief: >
+          The unique identifier of the AWS Bedrock Agent. [Agents](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html) assist users in accomplishing complex tasks and are capable of interacting with various knowledge bases, data sources, and APIs.
+        examples: ["WM5GWNK1AI"]
+      - id: aws.bedrock.data_source.id
```

Comment:
let's define a generic attribute for it, this is a shared concept across all genai systems

Reply:
Let me add that.